Mp3App/Features/Home/HomeViewController.swift

```diff
@@ -6,16 +6,32 @@
 //  Copyright © 2020 AnhLD. All rights reserved.
 //
 
-import Foundation
 import UIKit
 import Reusable
 import RxSwift
 import RxCocoa
 
 class HomeViewController: BaseViewController, StoryboardBased, ViewModelBased {
     var viewModel: HomeViewModel!
+    private let disposeBag = DisposeBag()
+    private let loadDataTrigger = PublishSubject<Void>()
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        loadData()
+    }
+    
+    private func loadData() {
+        bindViewModel()
```

Comment:
load data chỉ có thằng loadDataTrigger.onNext() thôi 
bindViewModel làm 1 việc hoàn toàn khác, sao lại đặt ở trong đó